src/docs/02_architecture_constraints.adoc

```diff
@@ -11,11 +10,17 @@
 
```

Comment:
React Native is not really a constraint but an option. 
I'm also missing SOLID and the use of POD

Reply:
It was a constraint until the decision to use React.
Now the constraint should be "normal" React.

Reply:
> 
> 
> React Native is not really a constraint but an option.
> I'm also missing SOLID and the use of POD

SOLID is the first constraint of the table. Nevertheless, I think that the pod and the use of linked data should be included.

Reply:
By the way, this shouldn't be merged to master, but to another branch. It will be better if we leave master branch for releases.

Reply:
Lets create a Documentation branch and merge this new one to master once we have it all done.